src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/viewmodel/TakeManagementViewModel.kt

```diff
@@ -82,7 +79,8 @@ class TakeManagementViewModel : ViewModel() {
             title = "$label $start"
             activeContent = it
             populateTakes(it)
-            getContentList(activeCollection)
+            // TODO
+//            getContentList(activeCollection)
```

Comment:
When will the TODO be resolved?  If not immediately, then at least make the TODO more descriptive.